test/e2e/framework/util.go

```diff
@@ -5192,18 +5192,25 @@ func WaitForPersistentVolumeClaimDeleted(c clientset.Interface, ns string, pvcNa
 	return fmt.Errorf("PersistentVolumeClaim %s is not removed from the system within %v", pvcName, timeout)
 }
 
-func GetClusterZones(c clientset.Interface) (sets.String, error) {
+func GetClusterZones(c clientset.Interface) (sets.String, error, error) {
```

Comment:
Why have two separate errors?

Reply:
Mabey does it return notfoundPartNodes names will be more clearer？

Reply:
I'd reuse the same returned error variable instead of adding a new one. The caller doesn't care what error it gets anyway

Reply:
@verult I have re pushed it ,how about this?